### ISSUE TITLE
Corrupted files after Trasnfer with DL

### ISSUE THREAD

**Issue by explit7** [bkw777/dlplus#5]
Hello Brian,
I have a strange behavior on my NEC PC-8201 when i transfer files with DesLink+ (actual Master branch 2.0.x)
When i start TS-DOS on my NEC and dl on my Debian Linux, anything seems to working fine.

I can load any .DO or .BA files from the current directory on my Linux Machine. 
When i try to open loaded files in basic - they don't
 run because corrupted.

I copied a file DSKLOG.DO from Linux PC to NEC and saved it back to Linux under different name.
And i see, that the size is different and first lines of file contains garbage. The rest of the file is normal.

DSKLOG.DO is the normal file
DSKLOGB.DO is broken file
Now i wonder, what is wrong on my configuration.

I am thankful for any help

Normal file
[DSKLOG.DO.TXT](https://github.com/bkw777/dlplus/files/10683313/DSKLOG.DO.TXT)

File after Transfer
[DSKLOGB.DO.TXT](https://github.com/bkw777/dlplus/files/10683316/DSKLOGB.DO.TXT)

**Comment by bkw777** [comment]
I can't duplicate.

I loaded the file to a PC-8300, saved back under a different name, and the only difference was the new copy has an EOF byte added to the end, which I think the main rom added, or possibly TS-DOS but I think not. The copy on the machine loaded into basic and ran without errors.

I'll try an 8201 too just for completeness but I doubt there will be any difference from an 8300.

I assume you've tried cold-resetting the NEC? The ram filesystem can become broken simply by software scribbling in memory, so if you haven't already I'd do a cold reset and try on a clean machine barring reinstalling ts-dos.

I don't recognize the junk. It would be interesting if we could tell if the junk came from the nec or the pc, like if it was a chunk of the nec main rom, or part of another file on the at the time.

Maybe also try a ram test app to see if you have any bad ram possibly, except I don't know where to find a ram test app for nec offhand. I just looked in livingm100sig but not thoroughly. Maybe web8201 has something. ... Well there's 3 or so here but they look like they're for m100 and may not work on nec https://www.web8201.net/files/body.asp?name=Utilities&url=Club100/Utilities
Similarly there's a RAMTST in m100s that looks like it includes machine language and probably only works on 100/102
https://github.com/LivingM100SIG/Living_M100SIG/blob/main/M100SIG/Lib-07-UTILITIES/RAMTST
There must be one out there somewhere but I'm not finding it.
